src/web/components/Dateline.tsx

```diff
@@ -17,18 +17,89 @@ const dateline = css`
     margin-bottom: 6px;
 `;
 
-const description = css`
-    ${visuallyHidden}
+const secondaryDateWidth = css`
+    ${from.leftCol} {
+        width: 90%;
+    }
 `;
 
+// Checkbox functionality
+const toggleClass = css`
+    input[type='checkbox'] {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        opacity: 0;
+        display: none;
+    }
+
+    input[type='checkbox']:checked ~ span ~ p {
+        max-height: 80px;
+    }
+`;
+
+const pCheckboxStyle = css`
+    p {
+        max-height: 0;
+        overflow: hidden;
+        transition: max-height 0.4s ease;
+    }
+`;
+
+const labelStyles = css`
```

Comment:
Yeah, I think it would great to move the `labelStyles` directly to a className attr on label